database/mongodb/mongodb.go

```diff
@@ -137,7 +137,16 @@ func (m *Mongo) Open(dsn string) (database.Driver, error) {
 	if err != nil {
 		return nil, err
 	}
-	maxLockingIntervals, err := parseInt(unknown.Get("x-advisory-lock-timout-interval"), DefaultLockTimeoutInterval)
+
+	lockTimeout := unknown.Get("x-advisory-lock-timeout-interval")
+
+	if lockTimeout == "" {
+		// The initial release had a typo for this argument but for backwards compatibility sake, we will keep supporting it.
+		lockTimeout = unknown.Get("x-advisory-lock-timout-interval")
```

Comment:
Thanks for preserving the old behavior! Let's return an error explaining that only one should be specified if both are specified.

Reply:
This fix isn't urgent so can wait until you're back from vacation. Or I can make the change (based off of your change). I'd like to support existing users in a backwards compatible manner but also eliminate any confusion if both parameters are specified.

Reply:
Hows that @dhui ?